### PR TITLE
feat(node-analyzer): bumped host-scanner to 0.7.0 to add container scanning support

### PR DESCRIPTION
**Pull request by michele-mangili**
## Checklist

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [ ] Title of the PR starts with type and scope, (e.g. feat(agent,node-analyzer,sysdig-deploy):)
- [ ] Chart Version bumped for the respective charts
- [ ] Variables are documented in the README.md (or README.tpl in some charts)
- [ ] Check GithubAction checks (like lint) to avoid merge-check stoppers
- [ ] All test files are added in the tests folder of their respective chart and have a "_test" suffix

Check Contribution guidelines in README.md for more insight.

### CODE REVIEW

File: charts/node-analyzer/README.md [hunk 1]
```diff
@@ -209,6 +209,9 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
...
+| `nodeAnalyzer.hostScanner.scanContainers.enabled`                           | Set to `true` to scan containers                                                                                                                                                    | `false`                                                                                                                                                             |
+| `nodeAnalyzer.hostScanner.scanContainers.dockerSocketPath`          | Specifies the path to docker socket                                                                                                                                                 | `unix:///var/run/docker.sock`                                                                                                                                                                  |
```
Comment (AlbertoBarba) [inline charts/node-analyzer/README.md:213]: I see this is commented in the values file. Is the default still valid?

File: charts/node-analyzer/README.md [hunk 2]
```diff
@@ -209,6 +209,9 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
...
+| `nodeAnalyzer.hostScanner.scanContainers.podmanSocketPath`          | Specifies the path to podman socket                                                                                                                                                 | `unix:///var/run/podman.sock`                                                                                                                                                                  |
```
Comment (AlbertoBarba) [inline charts/node-analyzer/README.md:214]: I see this is commented in the values file. Is the default still valid?

File: charts/node-analyzer/README.md (hunk 1 above)
Comment (michele-mangili) [reply charts/node-analyzer/README.md:213]: Yes, that's right. The `dockerSocketPath` is optional. When not provided explicitly we fallback to `unix:///var/run/docker.sock`.

File: charts/node-analyzer/README.md (hunk 2 above)
Comment (michele-mangili) [reply charts/node-analyzer/README.md:214]: Yes, see my comment above.

**Review by AlbertoBarba — APPROVED** [review]

**Pull request merged**